step.yml

```diff
@@ -124,6 +124,16 @@ inputs:
       value_options:
       - "yes"
       - "no"
+  - auto_merge: "yes"
+    opts:
+      category: Debug
+      title: Merge origin/master
+      summary:  Merge origin/master 
```

Comment:
Flag indicating if merged state should be checked out instead of just the source branch.